portfolio/src/main/java/com/google/sps/filters/ObjectifyWebFilter.java

```diff
@@ -17,6 +17,8 @@
 import javax.servlet.annotation.WebFilter;
 import com.googlecode.objectify.ObjectifyFilter;
 
-/** Filter needed by objectify. */
+/** Filter needed by objectify in order to clean up any thread-local transaction
```

Comment:
nit: start on the second line if it's a multi-line javadoc comment.
https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line